codalab_service.py

```diff
@@ -454,7 +468,7 @@ def resolve_env_vars(args):
             'CODALAB_HTTP_PORT': args.http_port,
             'CODALAB_VERSION': args.version,
             'CODALAB_WORKER_NETWORK_NAME': '%s-worker-network' % args.instance_name,
-            #'PATH': os.environ['PATH'],
+            'PATH': os.environ['PATH'],
```

Comment:
Oh yeah, thanks for re-adding this (#1161). We need to eventually include macOS on Travis tests.